detox/ios/Detox.xcodeproj/project.pbxproj

```diff
@@ -667,9 +667,9 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 3947679F1DBF985400D72256 /* Build configuration list for PBXNativeTarget "Detox" */;
 			buildPhases = (
+				394767941DBF985400D72256 /* Headers */,
 				394767921DBF985400D72256 /* Sources */,
 				394767931DBF985400D72256 /* Frameworks */,
-				394767941DBF985400D72256 /* Headers */,
```

Comment:
Thanks @ball-hayden! 🙂 
I just heard about this issue in Xcode 13.3, so I'm familiar with that.